rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -747,6 +748,38 @@ private void logCommitmentWithVotedFederation(BridgeEventLogger eventLogger, Fed
         logger.debug("[logCommitmentWithVotedFederation] Voted federation committed: {}", votedFederation.getAddress());
     }
 
+    @Override
+    public BtcTransaction createAndSetSvpFundTransactionWithoutSignatures(Coin spendableValueFromProposedFederation, Coin feePerKb, Wallet activeFederationWallet) throws InsufficientMoneyException {
```

Comment:
The fact that the method name has an `and` as part of it's name makes me think it's doing more than one thing and probably Uncle Bob does not approve. Should we split this into 2 separate methods?